### ISSUE TITLE
 Are the parameters tuned using the validation set?

### ISSUE THREAD

**Issue by abebe9849** [yrjia1015/THItoGene#1]
Thank you for publishing such a great repository!

https://github.com/yrjia1015/THItoGene/blob/a9536f3c061aeba19ae8371dbafde00a690df6f2/train.py#L30C37-L30C37

It seems that there is no validation set, but how did you decide on the structure of the model? (e.g. number of layers, etc.)

**Comment by yrjia1015** [comment]
Hi! Happy New Year!

In determining our model's parameters, we initially set them based on existing academic knowledge. We then refined these parameters through iterative testing and adjustments, using multiple datasets and cross-validation techniques. This process helped us find a balance between performance and scalability, resulting in an optimized parameter configuration for our model.

**Comment by abebe9849** [comment]
thanks! 

Are the data set for parameter tuning and the test set used for evaluation prepared separately?
Even if cross-validation is performed, I think it is necessary to prepare a separate data set from tuning. Since the amount of data is small, nested cross validation seems to be useful this time.

**Comment by yrjia1015** [comment]
Yes, thank you for your comment
